google/resource_bigquery_dataset_test.go

```diff
@@ -112,3 +159,110 @@ resource "google_bigquery_dataset" "test" {
   }
 }`, datasetID)
 }
+
+func testAccBigQueryDatasetWithOneAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
+  friendly_name               = "foo"
+  description                 = "This is a foo description"
```

Comment:
I'm fine with keeping these here, but I would also be fine with the test only setting required fields + access-related stuff